### PR TITLE
update to use the pipeline image for the linuxptp-daemon in ptp-operator

### PR DESCRIPTION
**Pull request by aneeshkp**

### CODE REVIEW

File: ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.12.yaml [hunk 1]
```diff
@@ -23,6 +27,8 @@ operator:
   substitutions:
   - pullspec: quay.io/openshift/origin-ptp-operator:4.12
     with: pipeline:ptp-operator
+  - pullspec: quay.io/openshift/origin-ptp:4.13
```
Comment (SchSeba) [inline ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.12.yaml:30]: this one is not right

File: ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.18.yaml [hunk 2]
```diff
@@ -25,6 +29,8 @@ operator:
   substitutions:
   - pullspec: quay.io/openshift/origin-ptp-operator:4.17
     with: pipeline:ptp-operator
+  - pullspec: quay.io/openshift/origin-ptp:4.17
```
Comment (SchSeba) [inline ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.18.yaml:32]: this one is not right

**Review by SchSeba — COMMENTED** [review]
also please run `CONTAINER_ENGINE=podman make ci-operator-config jobs openshift-image-mirror-mappings`

File: ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.12.yaml (hunk 1 above)
Comment (aneeshkp) [reply ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.12.yaml]: fixed

File: ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.18.yaml (hunk 2 above)
Comment (aneeshkp) [reply ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.18.yaml]: fixed

**Review by SchSeba — COMMENTED** [review]
/lgtm
/approve

**Comment by aneeshkp** [comment]
/pj-rehearse pull-ci-openshift-ptp-operator-release-4.16-security

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**